src/templates/parseScriptTemplates.ts

```diff
@@ -103,6 +104,7 @@ function parseScriptSetting(data: object, resources: IResources, variables: IVar
         name: getVariableValue(resources, variables, rawSetting.name),
         resourceType: rawSetting.resource,
         valueType: rawSetting.value,
+        description: rawSetting.help ? getResourceValue(resources, rawSetting.help) : undefined,
```

Comment:
Do we want to completely remove the description if there is no help dialogue?  Won't that get rid of whatever was normally there by default?

Reply:
Description was undefined in this object before my change, so I'm not removing anything

Reply:
Where was `Provide a 'Path'` coming from??

Reply:
https://github.com/Microsoft/vscode-azurefunctions/blob/ec9b71adbd5cb27d1206661837cbc7e72bef15d4/src/commands/createFunction/genericSteps/StringPromptStep.ts#L22

Reply:
Why not just set `rawSetting.description` instead of naming it `help`?  I get that it's more like a tooltip, but it ends up being equivalent to a quickpick's description property-- seems kinda confusing.

Reply:
I didn't choose the name `help`. This file is all about parsing templates that are defined externally and I have to use whatever structure that's given to us